src/environments/environment.model.ts

```diff
@@ -72,6 +72,9 @@ export interface Environment {
   // default viewType used for product listings
   defaultProductListingViewType: ViewType;
 
+  // default device type used for initial page responses
```

Comment:
Listing the 3 options would be nice here I guess.

Reply:
Sorry, I missed the fact that this was typed already. No need for additional documentation. :(